service/src/main/kotlin/fi/espoo/evaka/messaging/message/MessageQueries.kt

```diff
@@ -272,6 +273,74 @@ fun Database.Read.getMessage(id: UUID): Message {
         .single()
 }
 
+fun Database.Read.getCitizenReceivers(accountId: UUID): List<MessageAccount> {
+    val params = mapOf(
+        "accountId" to accountId,
+    )
+
+    // language=SQL
+    val sql = """
+WITH placement_ids AS (
+    SELECT pl.id AS id
+    FROM guardian g
```

Comment:
Now we take children whose guardian the person is, but I assume that the head of child should also be allowed to send a message even without guardianship (e.g. case sijoitusperhe). On the other hand you could ask whether we should exclude guardians who have been blocklisted on child's page (messages won't go to them but are they still allowed to send new messages?)

Reply:
I updated the logic:

* Fridge parent has the right to send a new message similarly to a guardian.
* A blocklisted recipient has no right to send a new message. (Also created a test that checks that this works as intended)